CHANGELOG.md

```diff
@@ -2,6 +2,22 @@
 
 ## master
 
+- PR [#111](https://github.com/palkan/logidze/pull/111) Global configuration for `:ignore_log_data` option ([@dmitrytsepelev][])
+
+Now it's possible to avoid loading `log_data` from the DB by default with
+
+```ruby
+Rails.application.config.logidze.select_ignore_log_data = true
```

Comment:
Can't see where we add `logidze` config to Rails.

Let's just use `Logidze.ignore_log_data_by_default = true` (yeah, and rename the option).